console-api/Cargo.toml

```diff
@@ -3,8 +3,8 @@ name = "console-api"
 version = "0.6.0"
 license = "MIT"
 edition = "2021"
-rust-version = "1.64.0"
-authors = ["Eliza Weisman <eliza@buoyant.io>", "Tokio Contributors <team@tokio.rs>",]
+rust-version = "1.70.0"
```

Comment:
why was it necessary to do a MSRV bump in this PR?

Reply:
Because clap v4.4.18 set the 1.70.0 as the MSRV.

Reply:
@hawkw we're already going to bump the MSRV to 1.74 in #515 (which fixes an issue with input on Windows).

@hi-rustin I propose that we focus on getting the ratatui PR in and then this PR won't need an MSRV bump at all. That will avoid having multiple bumps in close changes.

Reply:
I don't think 515 requires bumping the MSRV to 1.74.0. It also requires 1.70.0. See https://github.com/ratatui-org/ratatui/blob/efd1e476425477b0bce15e3dd96d5cdeb0e1174b/Cargo.toml#L21

But I am OK to wait for that PR to land.